### ISSUE TITLE
[Google Play] Your APK is only signed using the APK v1 signature scheme.

### ISSUE THREAD

**Issue by Oldes** [Gamua/Adobe-Runtime-Support#498]
## Problem Description

I noticed a new annoyance on Google's side when doing an update today:

> Warnings
> Your APK is only signed using the APK v1 signature scheme. This will fail to install in versions of Android equal or higher to 11 if the android:targetSdkVersion is set to 30 or higher. Please use apksigner to sign your APK or switch to the Android App Bundle to fix this.

As for now, only android:targetSdkVersion 29 is required so far, but anyway... @ajwfrost will it be addressed somehow by future Air SDK? Thanks a lot!

**Comment by ajwfrost** [comment]
Yes we're planning to switch to the Android App Bundle assuming we can resolve the issue with packaging multiple different resources, which I think uses jarsigner rather than apksigner.

**Comment by leossmith** [comment]
@ajwfrost Any rough estimates on when this is going to happen? Also I think that there shouldn't be a switch to aab but rather the option for it. We need .apk to be able to sideload / test / debug / or upload to other stores.

**Comment by ajwfrost** [comment]
@leossmith good point, yes. We can add this to the queue (probably APKv3 signatures) but it's not the most urgent of tasks currently, I think we can aim for early next year.
thanks

**Comment by Gokulv617** [comment]
@ajwfrost I have also faced the v1 signature scheme issue in one of my app while submitting to console. Looking forward to use Android App bundle soon.
